thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/template.hbs

```diff
@@ -0,0 +1,121 @@
+<div class="manage-alert-tune">
+
+  <div class="te-content-block">
+    <h4 class="te-alert-page__subtitle">Alert Performance by Application Group</h4>
+
+    <a class="te-pill-selectors__side-link te-pill-selectors__side-link--{{if viewTotals "active" "inactive"}}" {{action (toggle "viewTotals" this)}}>
```

Comment:
Let's extract the state of viewTotals to a variable called `isActive` in the controller or component and reference it here, since this logic is later repeated.

Reply:
Sure - Lets.